fastly/tls_custom_activation.go

```diff
@@ -112,17 +115,14 @@ func (c *Client) GetTLSActivation(i *GetTLSActivationInput) (*TLSActivation, err
 
 	path := ToSafeURL("tls", "activations", i.ID)
 
-	ro := &RequestOptions{
-		Headers: map[string]string{
-			"Accept": jsonapi.MediaType, // this is required otherwise the params don't work
-		},
-	}
+	requestOptions := CreateRequestOptions(i.Context)
+	requestOptions.Headers["Accept"] = jsonapi.MediaType // this is required otherwise the filters don't work
 
 	if i.Include != nil {
-		ro.Params = map[string]string{"include": *i.Include}
+		requestOptions.Params = map[string]string{"include": *i.Include}
```

Comment:
```suggestion
		requestOptions.Params["include"] = *i.Include
```

This would have overwritten the value for `Accept` which was inserted a few lines above. In case I missed any, check the 'git diff' output for any others instances of `map[string]string` which may indicate this same issue.

Reply:
fixed in `d2eef2a`

Reply:
and `f5ed9e1`